doc/en/components/scheduling/date-range-picker.md

```diff
@@ -0,0 +1,418 @@
+---
+title: {Platform} Date Range Picker Component - {ProductName}
+_description: Infragistics' {Platform} Date Range Picker allows the user to select a range of two dates from a calendar and set it in an input element.
+_keywords: {Platform} Date Range Picker, {ProductName}, Infragistics
+mentionedTypes: ["DateRangePicker"]
+---
+
+# {Platform} Date Range Picker Overview
+The {ProductName} Date Range Picker is a lightweight component that includes a text input and a calendar pop-up, allowing users to easily select start and end dates.  It is highly customizable to fit various application requirements, offering features such as date range restrictions, configurable date formats, and more.
+
+## Date Range Picker Example
+
+Below is a sample demonstrating the `DateRangePicker` component in action, where a calendar pop-up allows users to select start and end dates.
+
+`sample="/scheduling/date-range-picker/overview", height="500", alt="{Platform} Date Range Picker Overview Example"`
+
+### Getting Started
+
+<!-- WebComponents -->
+
+To start using the `DateRangePicker`, you first need to install the Ignite UI for Web Components by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+
+After that, you need to import the `DateRangePicker`, its necessary CSS, and register its module, as follows:
+
+```ts
+import { defineComponents, IgcDateRangePickerComponent } from 'igniteui-webcomponents';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+defineComponents(IgcDateRangePickerComponent);
+```
+
+<!-- end: WebComponents -->
+
+Now you can start with a basic configuration of the {Platform} `DateRangePicker`.
+
+```html
+<igc-date-range-picker anchor="hover-button">
+  Congrats you've hovered the button!
+</igc-date-range-picker>
+
+<igc-button id="hover-button">Hover me</igc-button>
+```
+
+For a complete introduction to the {ProductName}, read the [*Getting Started*](../general-getting-started.md) topic.
+
+## Usage
+
+The `DateRangePicker` allows users to select a start and end date either by choosing a date range from a dropdown/calendar pop-up or by typing directly into the input fields - one for the start date and one for the end date.
+
+The picker offers two modes for displaying date values: single input and two inputs. In single input mode, the field is read-only, and the date range cannot be edited by typing. In two inputs mode, however, users can edit the start and end dates by typing in separate input fields.
```

Comment:
```suggestion
The picker offers two modes for displaying date values: single input and two inputs. In single input mode, the field is non-editable, and the date range cannot be edited by typing. In two inputs mode, however, users can edit the start and end dates by typing in separate input fields.
```
After clarifying the states with the UX team, apparently the better term for this is "non-editable"